cogs/reminder.py

```diff
@@ -181,7 +181,7 @@ def __init__(self, bot):
                         await asyncio.sleep(1)
 
             conn.close()
-            await asyncio.sleep(60)
+            await asyncio.sleep(60)    # seconds
```

Comment:
In general, changes like this should be kept to a separate commit (as with the style change above) - the ideal case is for us to be able to find merge commits that identify exactly what has been introduced into master and check them out one by one to categorically see changes for each functionality. 

You don't have to change it, just a note :)